maven-archetype-quickstart/src/main/resources-filtered/archetype-resources/pom.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-#macro( compilerProperties $javaCompilerVersion )
+#macro( compilerProperties $jcv )
```

Comment:
Any reason why this variable has been renamed ? `javaCompilerVersion` seems quite easy to understand...

Reply:
There was the name clash between global and local variable.  wanted to make clear, that  it is obvious on first glance, what is local to macro, and what is global one.  See  https://github.com/apache/maven-archetypes/blob/master/maven-archetype-quickstart/src/main/resources-filtered/archetype-resources/pom.xml#L2
compare with full:  https://github.com/judovana/maven-archetypes/blob/defaultJdk8/maven-archetype-quickstart/src/main/resources-filtered/archetype-resources/pom.xml#L48